### ISSUE TITLE
Specmatic Stubs picks up 400 Examples and fails due to it.

### ISSUE THREAD

**Issue by barath121** [specmatic/specmatic#983]
**Description**
When starting specmatic stubs the stub file might has some invalid examples for 400 Responses.

**Steps to reproduce**
1. Create a api specs add example where some required fields are missing.
2. Start the stub.

**Expected behavior**
The stub should not take the examples as it does not use it.

**Expected behavior**
it failed with this error

```
In scenario "Courier Partners.. Response: Invalid."
  API: POST /___/internal/logistics/v2.0/company/(company_id:string)/application/(application_id:string)/courier-partners -> 400
  
    >> REQUEST.BODY.shipments[0].articles[0].id
  
       key named id in the example "Pune" was not found in the spec
```

**System Information:**
 - OS & version: MacOS 12.4
 - Specmatic version: 1.2.18(Node Wrapper) and 0.26.6(Python)
 - JDK version: 21.0.1

**Comment by yogeshnikam671** [comment]
I think, specmatic **does use the examples to create the stub**. 
Refer to : https://specmatic.in/getting_started.html#consumer-side---contract-as-a-stub--intelligent-service-virtualisation

Documentation says : `And these examples also serve as stub data when we run Spemcatic stub command`
@barath121

**Comment by harikrishnan83** [comment]
@barath121 Thanks a lot for reporting this.

We recently added support for using the examples in the OpenAPI spec as stub data also (as pointed out by @yogeshnikam671 above).

As you may recollect, even with static stub files, if a particular file is not in sync with the spec, only that file will not be loaded into the stub server. The stub server itself will be very much available and will load the spect itself despite that particular stub json (expectation) not being loaded.

Similarly, even with the inline named examples in the OpenAPI spec, if a particular example is invalid as per the schema, only that example is treated / loaded as an expectation. The stub server will continue operate as normal. So this should not prevent you from continuing to use your external expectations (both static and dynamic).

That said, you have made a valid point about the example that you may have written for testing a 4xx response, will always throw errors when being looked at from the point of view of stub server. We are working on the fix for this.

Meanwhile, as mentioned earlier, you can continue to use the Specmatic stub server as usual with external expectations.

**Comment by joelrosario** [comment]
Hi @barath121, upon deeper investigation, we determined that there was an issue with stubbing out a path which had a path parameter defined as an enum, which was the root cause of the stub file not getting picked up correctly, also resulting in a few other issues. This is now resolved and the fix has been released as part of 1.2.22.
